admin/sections/fileLoader.tsx

```diff
@@ -64,7 +54,7 @@ export const FileLoaderConfig = () => {
       <Form {...loaderForm}>
         <form
           className="space-y-4 mb-4"
-          onSubmit={loaderForm.handleSubmit(submitLoaderForm)}
+          onBlur={loaderForm.handleSubmit(submitLoaderForm)}
```

Comment:
what's the difference between `submitLoaderForm(loaderForm.getValues())` (used below)?

Reply:
the `onBlur={loaderForm.handleSubmit(submitLoaderForm)}` is applied to `onBlur` events of the all the fields while `submitLoaderForm(loaderForm.getValues())` is only applied to the current field.